recipes/biomechanical-analysis-framework/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "biomechanical-analysis-framework" %}
+{% set cxx_name = "libbiomechanical-analysis-framework" %}
+{% set version = "0.1.0" %}
+
+package:
+  # {{ name }}
+  name: biomechanical-analysis-framework
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ami-iit/biomechanical-analysis-framework/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: fb12cae8da4146c1bb39f7f71e5e95b43c8e8714a1065fa18dbb307c1e8f4d15
+
+build:
+  number: 0
+
+outputs:
+  # {{ cxx_name }}
+  - name: libbiomechanical-analysis-framework
+    script: build_cxx.sh  # [unix]
+    script: bld_cxx.bat  # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage(cxx_name, max_pin='x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib("c") }}
+        - {{ compiler('cxx') }}
+        - {{ cdt('mesa-libgl-devel') }}  # [linux]
+        - {{ cdt('mesa-dri-drivers') }}  # [linux]
+        - {{ cdt('libselinux') }}  # [linux]
+        - {{ cdt('libxdamage') }}  # [linux]
+        - {{ cdt('libxxf86vm') }}  # [linux]
+        - {{ cdt('libxext') }}     # [linux]
+        - cmake
+        - pkg-config
+        - ninja        
+      host:
+        - yarp
+        - idyntree
+        - spdlog
+        - catch2>=3.3.0
+        - manif
+        - libbipedal-locomotion-framework>=0.18.0
+        - xorg-libxfixes  # [linux]
+      run:
+        - libbipedal-locomotion-framework>=0.18.0
```

Comment:
This is no needed, as the `libbipedal-locomotion-framework` recipe has a `run_exports` in its recipe that will automatically make sure that the correct `run` dependency is added in the resulting recipe (see https://github.com/conda-forge/bipedal-locomotion-framework-feedstock/blob/28e0a93a0ba93841cf3215be2a2098ad5d5a25e5/recipe/meta.yaml#L26-L27 and https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements).